wsgi/odyssy/events/models.py

```diff
@@ -0,0 +1,28 @@
+from django.db import models
+
+
+class Event(models.Model):
+    start_time = models.DateField()
+    end_time = models.DateField()
+    event_date = models.DateField()
```

Comment:
I am confused with the use of `start_time` and `end_time`. As you have mentioned in a comment in #2 , I guessed that same will be applied here. This will be time period for which event description stays on-line. So, there must be some field where administrator mentions when event takes place. I am still confused, pleases suggest what to do @singhpratyush  sir ?